src/main/java/uk/gov/justice/digital/delius/jpa/standard/entity/Disability.java

```diff
@@ -28,6 +29,8 @@
     private LocalDate finishDate;
     @Column(name = "NOTES")
     private String notes;
+    @OneToMany(mappedBy = "disability", fetch = FetchType.LAZY)
+    private List<Provision> provisions;
```

Comment:
We had an interesting issue where it looked like the LAZY was not being honoured because it was a `mapped` relationship

Reply:
I'll have a look at the queries being generated.